helper_libs/og_themes.py

```
@@ -0,0 +1,615 @@
+        """Initialize the generator with image dimensions."""
+        self.width = width
+        self.height = height
+    
+    def _create_gradient_background(
+        self, 
+        color1: Tuple[int, int, int, int], 
+        color2: Tuple[int, int, int, int],
+        angle: float = 45.0
+    ) -> Image.Image:
+        """Create a smooth gradient background using NumPy for performance."""
+        img = Image.new('RGBA', (self.width, self.height))
```

Ruff (0.14.10)

Local variable `img` is assigned to but never used

Remove assignment to unused variable `img`

(F841)